src/lib/config/config.mjs

```diff
@@ -80,13 +80,16 @@ export function configParser(path) {
 
   logger.debug(`Reading config from ${path}.`, "config");
   const config = getConfig(path);
-  const argv = yargs(argvInput).options({
-    profile: {
-      default: "default",
-      alias: ["p"],
-      type: "string",
-    },
-  }).argv;
+  const argv = yargs(argvInput)
+    .options({
+      profile: {
+        default: "default",
+        alias: ["p"],
+        type: "string",
+      },
+    })
+    .help(false)
+    .version(false).argv;
```

Comment:
i think we should swap this out for yargs-parser instead

Reply:
That fails the test for some reason. I'll investigate.